jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/TaskCommonTest.java

```diff
@@ -131,6 +133,16 @@ public void testCheckDeprecatedUsage_extraDirectoryAndExtraDirectoriesConfigured
     }
   }
 
+  @Test
+  public void testCheckDeprecatedUsage_useCurrentTimestampConfigured() {
+    Mockito.when(containerParameters.getUseCurrentTimestamp()).thenReturn(true);
+    TaskCommon.checkDeprecatedUsage(jibExtension, logger);
+    Mockito.verify(logger, Mockito.times(1))
```

Comment:
The default behavior is `Mockito.times(1)`, so no need to specify that. The reason it is present in the extra directory test is that there are multiple depreated configurations and I wanted to ensure the user sees the message only once.